lib/liquid/tags/render.rb

```diff
@@ -43,19 +43,26 @@ def render_tag(context, output)
 
       context_variable_name = @alias_name || template_name.split('/').last
 
-      render_partial_func = ->(var) {
+      render_partial_func = ->(var, forloop) {
```

Comment:
Nit: I think the `forloop` argument is unnecessary. Since it's defined in the outer scope, it'll be available in the proc.

Reply:
Doesn't seem to like it considering it's nested inside the if statement and moving it up would mean putting the same guard around it anyway.